ui_framework/components/local_nav/_local_search.scss

```diff
@@ -28,6 +28,10 @@
   }
 }
 
+.kuiLocalLuceneSearchInput:focus {
+  padding-right: 13em;
+}
```

Comment:
Maybe add a comment to the styles noting that `13em is the width of 'Uses Lucene query syntax'` so it isn't just a magic number.